src/qibocal/auto/task.py

```diff
@@ -30,6 +32,14 @@ class Task:
     """Action object parsed from Runcard."""
     operation: Routine
 
+    @classmethod
+    def load(cls, path):
```

Comment:
Here and below
```suggestion
    def load(cls, path: Path):
```
or
```suggestion
    def load(cls, path: PathLike):
```

(irrelevant remark, but a few type hints are slightly helpful and almost for free)